lib/vdsm/virt/livemerge.py

```diff
@@ -364,12 +364,17 @@ def _start_commit(self, drive, job):
         try:
             base_target = drive.volume_target(job.base, actual_chain)
             top_target = drive.volume_target(job.top, actual_chain)
+            base_path = drive.volume_path(job.base)
+            top_path = drive.volume_path(job.top)
```

Comment:
We don't work with paths here, can be removed.

Reply:
Yes yes, this commit needs to be reworked.